app/helpers/arclight_helper.rb

```diff
@@ -225,4 +231,15 @@ def generic_should_render_field?(config_field, document, field)
   def generic_render_document_field_label(config_field, document, field: field_name)
     send(:"render_document_#{config_field}_label", document, field: field)
   end
+
+  private
+
+  def build_repository_link(document)
+    repository_path = Arclight::Repository.find_by(name: document.repository)&.slug if document.repository.present?
```

Comment:
```suggestion
    repository_path = document.repository_config
```
I'm wondering if this is equivalent?

Reply:
ah, yes. I see what I did now. this does work. updated and pushed change